README.md

```diff
@@ -221,3 +221,8 @@ Class | Method | HTTP request | Description
 ## Author
 
 support@fingerprint.com
```

Comment:
Consider adding a Support and feedback section consistent with the Node SDK: https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk#support-and-feedback

Reply:
Done!